### PR TITLE
Update munki.rb

### PR DESCRIPTION
**Pull request by Dials-Mavis**
##### Instructions
- Look for and complete the section relevant to your submission. Delete the others, including these `Instructions`.
- `{{cask_file}}` represents the cask file you’re submitting/editing (if applicable).
- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.
#### Editing an existing cask
- [*] Commit message includes cask’s name (and new version, if applicable).
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.

Updated $version string from 2.6.1.2684 to 2.7.0.2753

### CODE REVIEW

**Comment by fanquake** [comment]
You need to update the sha256sum and checkpoint values.

File: Casks/munki.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask 'munki' do
-  version '2.6.1.2684'
+  version '2.7.0.2753'
   sha256 'ad177547c2ebfb8332bfb929bb976b1a61d802d57a4a4d701a606506e48c2b21'
```
Comment (vitorgalvao) [inline Casks/munki.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by vitorgalvao** [comment]
Updating the `:checkpoint` value in `appcast` is required. You can find it with `curl --silent --compressed "{{appcast_url}}" | sed 's|<pubDate>[^<]*</pubDate>||g' | shasum --algorithm 256 | awk '{ print $1 }'`.

Alternatively, you may wish to use [`cask-repair`](https://github.com/vitorgalvao/tiny-scripts/blob/master/cask-repair). It can at once update the cask’s version (according to what you give it) taking care of updating both the cask’s `sha256` and the appcast’s `:checkpoint`, while still giving you the chance to perform other manual fixes.

You can easily install it with `brew install vitorgalvao/tiny-scripts/cask-repair`. `cask-repair --help` will show you some options, but as a whole the script will guide you through the update process.

You also need to fill the template. Please try again in another PR.

**Comment by vitorgalvao** [comment]
https://github.com/caskroom/homebrew-cask/pull/21239

**Pull request closed without merging**